### PR TITLE
Enabled tigera plugin for Kibana

### PR DESCRIPTION
**Pull request by Brian-McM**
## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [x] Milestone set according to targeted release.
- [x] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Review by doublek — APPROVED** [review]
LGTM

File: pkg/render/logstorage.go [hunk 1]
```diff
@@ -1136,6 +1136,10 @@ func (es elasticsearchComponent) kibanaCR() *kbv1.Kibana {
 			"rewriteBasePath": true,
 		},
 		"elasticsearch.ssl.certificateAuthorities": []string{"/usr/share/kibana/config/elasticsearch-certs/tls.crt"},
+		"tigera": map[string]interface{}{
+			"enabled":        true,
+			"licenseEdition": "enterpriseEdition",
```
Comment (tmjd) [inline pkg/render/logstorage.go:1141]: What is this licenseEdition? Should this be based on something read from the LicenseKey?

Comment (doublek) [reply pkg/render/logstorage.go:1141]: This is for updating the branding/logos in Kibana. If this is something that can be inferred from LicenseKeys then that would be better here.

Comment (doublek) [reply pkg/render/logstorage.go:1141]: Note that this is something that is in the management cluster and I am not fully aware of how we do licensing in Calico Cloud.

**Pull request merged**